File: PB - More Exercises JavaScript/6.0 Drawing Figures with Loops - More Exercises/07. Christmas Tree.js
function christmasTree(n) {
    //head
    for (let i = 0; i < 1; i++) {
        console.log(`${' '.repeat(n)} | ${' '.repeat(n)}`)
    }

    //body
    for (let i = 1; i <= n; i++) {
        console.log(`${' '.repeat(n - i)}${'*'.repeat(i)} | ${'*'.repeat(i)}`)
    }
}

// christmasTree(2)

File: PB - More Exercises GO/6.0 Drawing Figures with Loops - More Exercises/07. Christmas Tree.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	var n int
	fmt.Scan(&n)
	//head
	for i := 0; i < 1; i++ {
		fmt.Println(strings.Repeat(" ", n) + " | " + strings.Repeat(" ", n))
	}
	//body
	for i := 1; i <= n; i++ {
		fmt.Printf("%v%v | %v\n", strings.Repeat(" ", n-i), strings.Repeat("*", i), strings.Repeat("*", i))
	}

}

/*
name   :07. Christmas Tree
input  :3
output :
    |    
  * | *
 ** | **
*** | ***
*/
